_maps/configs/independent_bubble.json

```diff
@@ -3,12 +3,13 @@
 	"map_name": "Bubble-class Colonial Ship",
 	"map_short_name": "Bubble-class",
 	"map_path": "_maps/shuttles/shiptest/independent_bubble.dmm",
+	"limit": 1,
```

Comment:
you don't |need| to set this to one if you're adminspawning it but w/e

Reply:
yeahhh it's left over from when i was originally just going to put a spawn cap on it. i changed my mind.